app/src/components/SongInfoContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import {EnhancedComponent} from "./EnhancedComponent";
+import {IEnhancedComponentProps, IEnhancedComponentState} from "./EnhancedComponent";
+import "./SongInfoContainer.css";
+import {Image} from "./Image";
+
+class SongInfoContainer extends EnhancedComponent<ISongInfoContainerProps, ISongInfoContainerState> {
+    public static defaultProps: ISongInfoContainerProps = {
+        ...EnhancedComponent.defaultProps,
+        color: "#ffffff",
+    };
+
+    protected constructor(props: ISongInfoContainerProps) {
+        super(props);
+    }
+
+    public render() {
+        const albumPicture = "https://image.shutterstock.com/image-vector/minimal-abstract-black-sphere-design-600w-158384345.jpg";
+        const songDescription = "Song Name - Artist";
+        return (
+            <div className={"song-info-container"} style={{color: this.props.color}}>
+                <Image
+                    path={albumPicture}
+                    name={"Album"}
+                />
```

Comment:
I think the width and height for this can be larger than the default width you set.

For example, on YouTube, the size of the thumbnail is 270x150 when the icon size is 24x24. 
Since our icons are 30x30, we should make this approx 300x170px (16:9 aspect ratio - standard for YouTube thumbnail)

Accordingly you can also increase font size to 14px or 16px